app/controllers/api/v1/base_controller.rb

```diff
@@ -26,6 +26,6 @@ def internal_server_error(exception)
                else
                  { error: 'Internal Server Error' }
                end
-    render json: response, status: :internal_server_error
+    # render json: response, status: :internal_server_error
```

Comment:
why is this commented out?

Reply:
It was throwing an error that wasn't supposed to happen, but it seems some merge weirdness doubled up the index in the controller, which was causing it so i uncommented it.